src/syntax_text.rs

```diff
@@ -97,6 +96,7 @@ impl SyntaxText {
 
     pub fn for_each_chunk<F: FnMut(&str)>(&self, mut f: F) {
         enum Void {}
+        #[allow(clippy::unit_arg)]
```

Comment:
I personally don't like clippy-related attributes in the source. Could it be removed.

Moreover, in this case, this is clearly a false-positive, as `()` is spelled explicitly :)